libautoscoper/src/Video.cpp

```diff
@@ -194,6 +186,18 @@ int Video::create_background_image()
   return 1;
 }
 
+template <typename T> void Video::create_background_image_internal(TiffImage* tmp_img, unsigned int normalization_factor) {
```

Comment:
```suggestion
template <typename T> void Video::create_background_image_internal(TiffImage* tmp_img, unsigned int normalization_factor) {
```

Any suggestions for a better name for this function?